__tests__/app.test.js

```diff
@@ -268,6 +268,33 @@ describe("DELETE /api/comments/:comment_id", () => {
     })
 });
 
+describe("GET api/users", () => {
+    test("status 200, responds with an array of user objects", () => {
+        return request(app)
+        .get("/api/users")
+        .expect(200)
+        .then((response) => {
+            response.body.users.forEach((user) => {
+                expect(user).toMatchObject({
+                username: expect.any(String),
+                name: expect.any(String),
+                avatar_url: expect.any(String)
+            });
+            expect(response.body.users).toBeInstanceOf(Array);
+            expect(response.body.users).toHaveLength(4);
+            });
+        });
+    });
+    test("status 404: responds with 404 when endpoint is spelt incorrectly", () => {
+        return request(app)
+        .get("/api/usrs")
+        .expect(404)
+        .then((response) => {
+            expect(response.body.msg).toBe("Endpoint Not Found");
+        });
+    });
```

Comment:
You don't need to add a test for this. This should have been accounted for in previous tests  👍